library/core/src/internal_macros.rs

```diff
@@ -31,6 +31,7 @@ macro_rules! forward_ref_binop {
             type Output = <$t as $imp<$u>>::Output;
 
             #[inline]
+            #[track_caller]
```

Comment:
Hmm, do we really want this on *everything* rather than just `/` and `%`?

I guess if they overflow in debug it might be helpful?  Let's see whether perf says this comes out painful...

Reply:
> do we really want this on everything rather than just / and %?

Simply adding it for ﻿`/` and `﻿%` should be sufficient. I will change that after perf reported.